### PR TITLE
Add pause button 

### LINKED ISSUE #20: Feature: pause button
Pointillism activity takes the live camera image and scatters it on the display.  It can be hard to get the image to look the way you want.

Add a pause button that pauses the scattering.  The button should change from the media pause icon to the media play icon.  Pressing it again should resume the scattering.

Sugargame test activity has useful pause feature.

### PR DESCRIPTION
**Pull request by ayushnawal**
PR includes:
* Fix PyGI warning
* Add pause button 

Fixes #20

### CODE REVIEW

**Comment by quozl** [comment]
Thanks.  Pause flag is set but not used in the main loop.  Have you missed something?

**Comment by ayushnawal** [comment]
I am not sure though, is this the fix you were suggesting?

**Comment by quozl** [comment]
I was not suggesting any specific fix, but rather pointing out that your earlier patch 0382ad3 added a pause button and a paused flag, but did not use the flag to pause the pointillism effect.

Looking at 006b561, I can't see why you are adding this code or what it does.  It refers to undefined variables and looks nothing like what is already in the file.

Contributing to this, your commit messages are too brief to explain what you did and why you did it.  See [Making commits](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#making-commits) for our standing advice on commit messages.

File: activity.py [hunk 1]
```diff
@@ -54,7 +54,8 @@ def __init__(self, handle):
         self.radio_dos = 12
 
         self.actividad = puntillism.Puntillism(self)
-
+        self.paused = False 
+        self.game = puntillism.Puntillism(self)
```
Comment (srevinsaju) [inline activity.py]: You are creating a new instance of Puntillism Object here, consider removing it

File: puntillism.py [hunk 2]
```diff
@@ -100,6 +104,30 @@ def run(self):
             events = pygame.event.get()
             self.read_events(events, screen, font, x_s, y_s)
 
+            if not self.paused:
```
Comment (srevinsaju) [inline puntillism.py]: Should move this to outer loop, should not depend on `poner_radio` to be changed

File: puntillism.py [hunk 3]
```diff
@@ -24,9 +24,9 @@
 
 from gettext import gettext as _
 import random
-from gi.repository import Gtk
 import gi
 gi.require_version('Gtk', '3.0')
```
Comment (srevinsaju) [inline puntillism.py]: Consider removing this too. There is the need of only one instance of gi.require_version, and that is on activity.py

File: activity.py [hunk 4]
```diff
@@ -144,6 +154,20 @@ def build_toolbar(self):
         self.set_toolbar_box(toolbar_box)
         toolbar_box.show_all()
 
+    def _pause_play_cb(self, button):
+        # Pause or unpause the game.
+        self.paused = not self.paused
+        self.game.set_paused(self.paused)
```
Comment (srevinsaju) [inline activity.py]: Cant see where `set_paused` is defined

**Pull request closed without merging**